lib/core/versions/0.9.0/models/PublicKeyModel.ts

```diff
@@ -1,4 +1,4 @@
-import PublicKeyPurpose from "../PublicKeyPurpose";
+import PublicKeyPurpose from '../PublicKeyPurpose';
```

Comment:
This was a pre-existing lint error fixed by `npm run lint`

Reply:
Thank you!